File: hackerrank/warmup/loveletter.cs
using System;

public class loveletter{
    
    public static bool isPalindrome(String value){
        bool isPalin = true;
    
        for(int i = 0; i < value.Length/2;i++){
            if(value[i] != value[value.Length-i-1]){
                isPalin = false;
            }
        }

        return isPalin;
    }
    
    public static void Main(){
        String val = Console.ReadLine();
        Console.WriteLine(isPalindrome(val));
    }
}

File: hackerrank/warmup/loveletter.rb


def isPalindrome(value)
  puts value.length
  length = value.length/2
  isPalin = true;
  for i in 0..length
      if value[i] != value[value.length-i-1]
        return false
      end
  end
  return isPalin
end

value = gets.chomp

puts isPalindrome(value)
